notebooks/125-torchvision-zoo-to-openvino/125-lraspp-segmentation.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        traced_model = torch.jit.trace(model, dummy_input, strict=False)
</pre><p>if you move on nightly or prerelease, it is not necessary anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Set requirement `pip install -q --pre "openvino>2023.1.0"`